cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -2030,6 +2030,10 @@ msgstr "Bydd angen i chi ddarparu tystiolaeth o’r wybodaeth ariannol yr ydych
 msgid "We’ll give you the CLA phone number when you submit your details below."
 msgstr "Byddwn yn rhoi rhif ffôn CLA i chi pan fyddwch yn cyflwyno eich manylion isod."
 
+#: cla_public/templates/contact.html:114
+msgid "You can enter up to %(cookie_settings_url)s characters"
```

Comment:
Judging solely based on the variable name, is `cookie_settings_url` correctly referencing what we want here?

Reply:
No...  good spot.  Copy-paste error.  😱

Reply:
Fixed